public/apps/account/test/__snapshots__/log-out-button.test.tsx.snap

```diff
@@ -1,11 +1,50 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Account menu - Log out button renders renders when auth type is MultiAuth: basicauth 1`] = `
+<div>
+  <EuiButtonEmpty
+    color="danger"
+    data-test-subj="log-out-3"
+    onClick={[Function]}
+    size="xs"
+  >
+    Log out
+  </EuiButtonEmpty>
+</div>
+`;
+
+exports[`Account menu - Log out button renders renders when auth type is MultiAuth: openid 1`] = `
+<div>
+  <EuiButtonEmpty
+    color="danger"
+    data-test-subj="log-out-2"
+    onClick={[Function]}
+    size="xs"
+  >
+    Log out
+  </EuiButtonEmpty>
+</div>
+`;
+
+exports[`Account menu - Log out button renders renders when auth type is MultiAuth: saml 1`] = `
+<div>
+  <EuiButtonEmpty
+    color="danger"
+    data-test-subj="log-out-1"
+    onClick={[Function]}
+    size="xs"
+  >
+    Log out
+  </EuiButtonEmpty>
+</div>
+`;
+
 exports[`Account menu - Log out button renders renders when auth type is OpenId 1`] = `
 <div>
   <EuiButtonEmpty
     color="danger"
     data-test-subj="log-out-2"
-    href="/auth/logout"
+    onClick={[Function]}
```

Comment:
Where is `Function` defined.

Reply:
This is only test snapshot. Line 47 is for ` onClick={() => externalLogout(props.http, OPENID_AUTH_LOGOUT)}`